test/new-e2e/system-probe/config/platforms.json

```diff
@@ -6,31 +6,31 @@
         "ubuntu_20.04": "master/ubuntu-20-x86_64.qcow2.xz",
         "ubuntu_22.04": "master/ubuntu-22-x86_64.qcow2.xz",
         "ubuntu_23.10": "master/ubuntu-23-x86_64.qcow2.xz",
-        "amzn_4.14":    "master/amzn2-kvm-2.0-x86_64-4.14.qcow2.xz",
```

Comment:
I agree with this, but it feels like a free change nevertheless

Reply:
I think this one was the autoformatter + some merge changes. In my original branch it was like this.